analysis/01_consensus_peaks/density_plot_promoters.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "DNA-binding at promoters"
+output: html_document
+editor_options:
+  chunk_output_type: console
+---
+
+
+```{r setup, include=FALSE}
+options(stringsAsFactors = F)
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(GenomicRanges)
+library(GenomicFeatures)
+library(rtracklayer)
+library(ggrepel)
+# BiocManager::install("Rsubread", version = 3.8)
+library(Rsubread)
+```
+
+
+### Goals of this code:
+1) Make a density plot of number of transcription factors bound at a given promoter.
+2) What promoters are never bound by TFs?
+3) Are there any DNA binding proteins that did not overlap at any promoter?
+ 
+
+Question #1: Density plot of TFs bound at promoters
```

Comment:
Here we are going to make density plots for the number of DBPs (DNA binding proteins) at a given promoter.